functions/Test-DbaPowerPlan.ps1

```diff
@@ -59,71 +59,116 @@ Test-DbaPowerPlan -ComputerName sqlserver2014a -Detailed
 To return detailed information Power Plans
 
 #>
-	[CmdletBinding(SupportsShouldProcess = $true)]
-	[OutputType([System.Collections.ArrayList])]
-	Param (
-		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+	param (
+		[parameter(ValueFromPipeline = $true)]
 		[Alias("ServerInstance", "SqlInstance", "SqlServer")]
-		[string[]]$ComputerName,
+		[string[]]$ComputerName = $env:COMPUTERNAME,
+		[PSCredential][System.Management.Automation.CredentialAttribute()]$Credential,
 		[string]$CustomPowerPlan,
-		[switch]$Detailed
+		[switch]$Detailed,
+		[switch]$Silent
 	)
-
-	BEGIN
+	
+	begin
 	{
+		If ($Detailed)
+		{
+			Write-Warning "Detailed is deprecated and will be removed in dbatools 1.0"
```

Comment:
Thank you. I removed the once, though, i dont wanna waste some metadata on that it.